package.json

```diff
@@ -55,7 +55,7 @@
     "build:props": "lerna run build:props --ignore @twilio-paste/website --concurrency 2",
     "pre-push": "concurrently \"yarn:lint\" \"yarn:test\" \"yarn:prettier\" \"yarn:type-check\"",
     "prerelease": "yarn build && yarn lint && yarn test",
-    "release": "yarn prerelease && yarn changeset publish",
+    "release": "yarn changeset publish",
```

Comment:
Can I get some info around this change please?

Reply:
Yup, so here's my logic. On merge to main two things need to happen:

1. Either version or publish on the packages
2. Run chromatic to create a new main branch baseline

To do either, you must build. There shouldn't be a need to build more than once, that's wasteful. If the build fails neither chromatic nor versioning is worth running.

If the build succeeds, but versioning fails, we still need a new chromatic baseline otherwise active branches are out of syn

If the build succeeds, but chromatic fails, we still need versioning to run.

So to keep all three independent, the build needs to run once, and be shared. The problem is if the release must run `prerelease` that runs the build again. The easiest way is to make release independent, and make prerelease the first job to both versioning and chromatic.